base/sparse/cholmod.jl

```diff
@@ -1750,17 +1760,27 @@ end
 
 det(L::Factor) = exp(logdet(L))
 
-function isposdef(A::SparseMatrixCSC{<:VTypes,SuiteSparse_long})
```

Comment:
Yes and I believe that https://github.com/JuliaLang/julia/pull/22345/files#diff-f75d450eb1491e3f787ed1ae6b40424fR378 should be exercising that.